sponsor-dapp/src/utils/FormattingUtils.js

```diff
@@ -17,3 +17,22 @@ export function formatWei(num, web3) {
   // See https://github.com/ethereum/web3.js/issues/1777.
   return web3.utils.fromWei(num.toString());
 }
+
+// Formats the input to round to decimalPlaces number of decimals.
+export function formatWithMaxDecimals(num, decimalPlaces, roundUp) {
+  const fullPrecisionFloat = Number.parseFloat(num);
+  let fixedPrecisionFloat = Number.parseFloat(fullPrecisionFloat.toFixed(decimalPlaces));
```

Comment:
Wdyt about using the other BigNumber library instead, that takes a rounding mode to its `toFixed` function? (Just a thought, this is totally fine as well)

Reply:
Yup, I think that's way simpler. Fixed.